rafthttp/transport.go

```diff
@@ -110,10 +116,26 @@ type Transport struct {
 	prober probing.Prober
 }
 
-func (t *Transport) Start() {
+func (t *Transport) Start() error {
+	var err error
+	// stream uses long term connection, and maintains heartbeat on the connection.
+	// It expects underlying connection returns error when taking long time to read and write,
+	// which helps to promptly find out about broken connection.
+	t.streamRt, err = transport.NewTimeoutTransport(t.TLSInfo, t.DialTimeout, ConnReadTimeout, ConnWriteTimeout)
+	if err != nil {
+		return err
+	}
+	// pipeline sends one message in one request.
+	// Because the size of raft snapshot message could be big, it may take long
```

Comment:
Line129 only states some facts, then what? What do you actually want to explain? I failed to see the relationship between your comment and the code below it.